site/assets/scss/_sidebar.scss

```diff
@@ -36,18 +53,16 @@
   padding: .1875rem .5rem;
   margin-top: .125rem;
   margin-left: 1.375rem; // Boosted mod: changed value
-  color: rgba($black, .65);
+  color: var(--bs-body-color);
   text-decoration: if($link-decoration == none, null, none);
 
-  // Boosted mod: no special hover, focus and active rules
+  // Boosted mod: only rule for hover
+  &:hover {
+    color: var(--bs-link-hover-color);
+  }
```

Comment:
Not sure if we should have an override here. If we accept the previous change, it is mandatory. Otherwise, we can remove the color attributes from `.bd-links-link` IMO

Reply:
OK need to Boosted mod line 56

Reply:
Simplified as mentioned via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/8a0e7b0d7e54b18aea9a2d2b597847267cf9e434